pkg/apis/extensions/v1beta1/register.go

```diff
@@ -60,6 +58,7 @@ func addKnownTypes() {
 		&ListOptions{},
 		&ConfigMap{},
 		&ConfigMapList{},
+		&v1.DeleteOptions{},
```

Comment:
Remember, delete options are query based and have no innate type meta.
They exist as a schema tied to every version and group.